src/codeu/chat/util/CommandTokenizer.java

```diff
@@ -0,0 +1,67 @@
+package codeu.chat.util;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommandTokenizer {
+
+	private String source;
+	private int currentIndex = 0;
+
+	public CommandTokenizer(String source) {
```

Comment:
this is another small detail, but for readability purposes, I think it'd be nice to put this function after getCommand() so it makes sense reading this file from top to bottom